src/inverseRobot.js

```diff
@@ -8,6 +8,18 @@
 
 function inverseRobot(robot) {
   // write code here
+  const inverse = {};
+
+  for (const key in robot) {
```

Comment:
Consider using 'let' instead of 'const' for variable 'key' since the value may change in each iteration.